lobby-db/README.md

```diff
@@ -114,3 +114,22 @@ Typical commands:
 - https://flywaydb.org/documentation/gradle/migrate
 - https://github.com/triplea-game/lobby
 
+## Docker
+
+There is a Dockerfile in this project for building a lobby database image that can be used for development/testing.
+
+### Build
+
+Build the lobby database image using the following command (run from this directory):
+
+```
+$ docker build --tag triplea/lobby-db:latest .
+```
+
+### Run
```

Comment:
Suggestion: A "connecting to DB" section might be useful to have, most notably with which creds to use (just to make it easy).

Reply:
#3534